ComponentKit/Utilities/CKComponentDelegateAttribute.mm

```diff
@@ -0,0 +1,143 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#import "CKComponentDelegateAttribute.h"
+
+#import <vector>
+#import <streambuf>
+#import <objc/runtime.h>
+
+#import "CKAssert.h"
+#import "CKInternalHelpers.h"
+#import "CKMutex.h"
+#import "CKComponentViewInterface.h"
+#import "CKComponentSubclass.h"
+
+std::string identifierFromDefinition(const CKComponentDelegateAttributeDefinition& first)
```

Comment:
Silly nit: we use weird spacing, and might as well be consistent. `const CKFoo &foo`.